WordPress/Classes/ViewRelated/Pages/PageListViewController.swift

```diff
@@ -345,7 +345,16 @@ import WordPressComAnalytics
     func configureCell(cell: BasePageListCell, atIndexPath indexPath: NSIndexPath) {
         cell.accessoryType = .None
         cell.selectionStyle = .None
-        cell.delegate = self
+        
+        if cell.reuseIdentifier == self.dynamicType.pageCellIdentifier {
+            cell.onAction = { [weak self] (cell: BasePageListCell, button: UIButton, provider: WPPostContentViewProvider) in
```

Comment:
I wouldn't bother specifying types here – this seems cleaner to me?

```
cell.onAction = { [weak self] cell, button, provider in
    self?.handleMenuAction(fromCell: cell, fromButton: button, forProvider: provider)
}
```

(and the same with the block below)